src/amo/components/AddonDetail.js

```diff
@@ -110,4 +116,7 @@ class AddonDetail extends React.Component {
   }
 }
 
-export default translate({ withRef: true })(AddonDetail);
+export default compose(
+  translate({ withRef: true }),
+  withInstallHelpers({ src: 'dp-btn-primary' }),
```

Comment:
I'm not sure why, that's what we expect in GA though so I don't think we can change it.